zebrad/src/commands.rs

```diff
@@ -50,28 +50,9 @@ pub enum ZebradCmd {
 }
 
 impl ZebradCmd {
-    /// Returns true if this command sends output to stdout.
-    ///
-    /// For example, `Generate` sends a default config file to stdout.
-    ///
-    /// Usage note: `abscissa_core::EntryPoint` stores an `Option<ZerbradCmd>`.
-    /// If the command is `None`, then abscissa writes zebrad usage information
-    /// to stdout.
-    pub(crate) fn uses_stdout(&self) -> bool {
-        match self {
-            // List all the commands, so new commands have to make a choice here
-            Generate(_) | Help(_) | Revhex(_) | Version(_) => true,
-            Seed(_) | Start(_) => false,
-        }
-    }
-
```

Comment:
In the new logic we only change the tracing level from `warn` if we're running a server, so this function (which is the inverse of `is_server`) is no longer used.